errors.toml

```diff
@@ -105,6 +120,7 @@ zh = "未提供身份验证信息"
 # ErrorAuth is used in SSO communication failures
 de = "authentifizierungsfehler"
 en-us = "authentication failure"
+es-es = "fallo de autentificacion"
```

Comment:
I'm told this should be 'autentificación'

Reply:
fixed.